02-library/cs506/sim.py

```
@@ -1,13 +1,55 @@
-def euclidean_dist(x, y):
-    raise NotImplementedError()
 
+def euclidean_dist(x, y):
+    #raise NotImplementedError()
+    if (len(x) != len(y)):
+        return "wrong dimensions"
```

better to raise an error than return a value that is of an unexpected type.